test/CSSTransition-test.js

```diff
@@ -112,6 +112,70 @@ describe('CSSTransition', () => {
         }
       });
     });
+
+    describe('isAppearing', () => {
```

Comment:
it was also a bit confusing that this didn't work for the false part (because this pattern did work for the 'true' part):

```
it('should be false', done => {

        instance = tsp(
          <CSSTransition
            timeout={10}
            classNames="test"
            in={true}
          >
            <div/>
          </CSSTransition>
        )
          .render();


        instance.props({
          /* no in:true goes here..*/
         onEnter() ....
```